### ISSUE TITLE
Support for iOS 10 with excluded archs for simulator

### ISSUE THREAD

**Issue by madiguzel** [SwiftyTesseract/SwiftyTesseract#85]
Hi,

First of all, many thanks for this useful framework. I'm trying to add this framework in the podspec of my framework which has to support for iOS 10 as below:

....
s.ios.deployment_target = '10.0'
s.dependency 'SwiftyTesseract'
s.pod_target_xcconfig      = { "EXCLUDED_ARCHS[sdk=iphonesimulator*]" => "arm64" }
s.user_target_xcconfig     = { "EXCLUDED_ARCHS[sdk=iphonesimulator*]" => "arm64" }
....

When I run pod lint, it uses SwiftyTesseract 2.2.3 version and doesn't build because of iphonesimulator architecture problem. Is it possible to edit the framework to support iOS 10 or how can I use 2.2.3 version excluding iphonesimulator arm64 archs?

Thank you.

### Checklist
- [] I've read the [Contribution Guidelines](https://github.com/SwiftyTesseract/SwiftyTesseract/blob/master/CONTRIBUTING.md)
- [] I've read the [Code of Conduct](https://github.com/SwiftyTesseract/SwiftyTesseract/blob/master/CODE_OF_CONDUCT.md).

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Steven0351** [comment]
Thanks for enjoying the library, unfortunately in this situation I've made the decision to no longer actively support iOS version less than 11.0 and I no longer actively support CocoaPods as a dependency manager. When I released version 3.1.3 of the library (the last to support Cocoapods), I added the same thing you have in your podspec to fix the issue:
```
s.pod_target_xcconfig = { "EXCLUDED_ARCHS[sdk=iphonesimulator*]" => "arm64" }
s.user_target_xcconfig = { "EXCLUDED_ARCHS[sdk=iphonesimulator*]" => "arm64" }
```
You may be better off forking the project from version 2.2.3, adding that to the podspec, and maintaining your own fork of it that way.

**Comment by madiguzel** [comment]
Thank you for quick reply. I'll give a try :)
